src/student_success_tool/analysis/pdp/features/student.py

```diff
@@ -84,3 +87,9 @@ def student_program_of_study_changed_first_year(
     df: pd.DataFrame, *, term_col: str, year_col: str
 ) -> pd.Series:
     return df[term_col].ne(df[year_col]).astype("boolean")
+
+
+def diff_gpa_year_1_to_term_1(
```

Comment:
nit: This naming reads as backwards to me, should it be `diff_gpa_term_1_to_year_1` instead?

Reply:
Actually, I'd really like these changes in the develop branch; going to merge, and then will submit a follow-up PR with the tweaked feature/function name. Hope that's fine by you!